src/Security/Voter/RealmSigningLogVoter.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of letswifi; a system for easy eduroam device enrollment
+ * Copyright: 2023, Paul Dekkers, SURF <paul.dekkers@surf.nl>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+namespace App\Security\Voter;
+
+use App\Entity\Contact;
+use App\Entity\RealmSigningLog;
+use LogicException;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Authorization\Voter\Voter;
+
+use function assert;
+
+class RealmSigningLogVoter extends Voter
+{
+    public const EDIT = 'edit';
+
+    protected function supports(string $attribute, mixed $subject): bool
+    {
+        // if the attribute isn't one we support, return false
+        return $attribute === self::EDIT;
```

Comment:
Wellicht wel even checken of subject ook een RealmSigningLog is, anders komt elke security check hierlangs.